### PR TITLE
Optional Settings

### PR DESCRIPTION
**Pull request by spektra2147**

### CODE REVIEW

File: addons/default/visiosoft/advs-module/resources/views/list/partials/ads.twig [hunk 1]
```diff
@@ -5,16 +5,13 @@
     <table class="w-100 text-center">
         <thead>
         <tr class="text-white">
-            <th class="text-nowrap" colspan="2">{{ trans('visiosoft.module.advs::field.ad_title') }}</th>
+            <th class="text-nowrap" colspan="3">{{ trans('visiosoft.module.advs::field.ad_title') }}</th>
```
Comment (diashalabi) [inline addons/default/visiosoft/advs-module/resources/views/list/partials/ads.twig:8]: why make it 3? it should extend only 2 columns, the photo and the title columns

File: addons/default/visiosoft/advs-module/resources/views/list/partials/ads.twig [hunk 2]
```diff
@@ -5,16 +5,13 @@
...
             {% if listLocation %}
                 <th class="text-nowrap">
                     {{ trans('visiosoft.module.advs::field.city.name') }}
                     / {{ trans('visiosoft.module.advs::field.district.name') }}
                 </th>
             {% endif %}
-            {% if setting_value('visiosoft.module.advs::market_place') %}
-                <th class="text-nowrap">{{ trans('visiosoft.module.advs::field.listing_date') }}</th>
-            {% endif %}
```
Comment (diashalabi) [inline addons/default/visiosoft/advs-module/resources/views/list/partials/ads.twig:17]: why remove the header?

**Review by diashalabi — CHANGES_REQUESTED** [review]

File: addons/default/visiosoft/advs-module/resources/views/list/partials/ads.twig (hunk 2 above)
Comment (spektra2147) [reply addons/default/visiosoft/advs-module/resources/views/list/partials/ads.twig:17]: @diashalabi #3835 -12 check Gitlab issue

File: addons/default/visiosoft/advs-module/resources/views/list/partials/ads.twig (hunk 1 above)
Comment (spektra2147) [reply addons/default/visiosoft/advs-module/resources/views/list/partials/ads.twig:8]: @diashalabi #3835 -12 check Gitlab issue

Comment (diashalabi) [reply addons/default/visiosoft/advs-module/resources/views/list/partials/ads.twig:8]: if one customer wants to remove it, that doesn't mean all customers want to remove it

**Review by fatihalp — APPROVED** [review]

**Review by diashalabi — APPROVED** [review]

**Pull request merged**